app/uk/gov/hmrc/individualsbenefitsandcreditsapi/connectors/IfConnector.scala

```diff
@@ -66,10 +63,9 @@ class IfConnector @Inject()(servicesConfig: ServicesConfig, http: HttpClient)(
 
   private def header(extraHeaders: (String, String)*)(
       implicit hc: HeaderCarrier) =
-    // The correlationId should be passed in by the caller and will already be present in hc
     hc.copy(
         authorization =
-          Some(Authorization(s"Bearer $integrationFrameworkBearerToken")))
+          Option(Authorization(s"Bearer $integrationFrameworkBearerToken")))
```

Comment:
Options are generally preferable to directly using Ssome - this example won't fail in either case because the token is encoded in a string. Authorization will always be non-null, so Option should always pass.